content/en/blog/2024/gateway-mesh-ga/index.md

```diff
@@ -0,0 +1,98 @@
+* Optionally, we can modify the request. Here, we add a header.
+* Finally, we select a destination for the request. In this example, we are picking between two versions of our application.
+
+For more details, see [Istio's traffic routing internals](/docs/ops/configuration/traffic-management/traffic-routing/) and [Gateway API's Service documentation](https://gateway-api.sigs.k8s.io/mesh/service-facets/).
+
+## Which API should I use?
+
+With overlapping responsibilities (and names!), picking which APIs to use can be a bit confusing.
+
+Here is the breakdown:
+
+| API Name     | Object Types                                                                                                                          | Status                            | Recommendation                                                             |
+|--------------|---------------------------------------------------------------------------------------------------------------------------------------|-----------------------------------|----------------------------------------------------------------------------|
+| Istio APIs   | [Virtual Service](/docs/reference/config/networking/virtual-service/), [Gateway](/docs/reference/config/networking/gateway/)          | `v1` in Istio 1.22 (2024)         | Use for existing deployments, or where advanced features are needed        |
+| Gateway APIs | [HTTPRoute](https://gateway-api.sigs.k8s.io/api-types/httproute/), [Gateway](https://gateway-api.sigs.k8s.io/api-types/gateway/), ... | Stable in Gateway API v1.0 (2023) | Use for new deployments, in particular with [ambient mode](/docs/ambient/) |
+| Ingress API  | [Ingress](https://kubernetes.io/docs/concepts/services-networking/ingress)                                                            | Stable in Kubernetes v1.19 (2020) | Use only for legacy deployments                                            |
+
+You may wonder, given the above, why were Istio APIs [promoted to `v1`](/blog/2024/v1-apis) concurrently?
+This was part of an effort to accurate categorize the *stability* of the APIs.
+While we view Gateway API as the future (and present!) of traffic routing APIs, our existing APIs are here to stay for the long run, with full compatibility.
+This mirrors Kubernete's approach with [`Ingress`](https://kubernetes.io/docs/concepts/services-networking/ingress), which was promoted to `v1` while directing future work towards the Gateway API.
+
+## Community
+
+This stability graduation represents the culmination of countless hours of work and collaboration across an incredible diverse group.
+It is incredible to look at the [diverse list of organizations](https://gateway-api.sigs.k8s.io/implementations/) involved in the API and look back at how far we have come.
+
+A special thanks goes out to my [co-leads on the effort](https://gateway-api.sigs.k8s.io/mesh/gamma/): Flynn, Keith Mattix, and Mike Morris, as well as the countless others involved.
```

Comment:
```suggestion
A special thanks goes out to [my co-leads on the effort](https://gateway-api.sigs.k8s.io/mesh/gamma/): Flynn, Keith Mattix, and Mike Morris, as well as the countless others involved.
```

nit: give Flynn another name, please

Reply:
It is my understanding that is their chosen representation (see https://kubernetes.io/blog/2024/05/09/gateway-api-v1-1/, their github, etc). I can double check with them.